### DISCUSSION TITLE
Is there any reason to use multiple concepts if you're just going to use [filewords] for all of them, and not using classification images?

### DISCUSSION THREAD

**Discussion by morphinapg** [d8ahazard/sd_dreambooth_extension#1226] [Q&A]
Like, what specifically does it do? Is it actually affecting the training optimization in some way, or is it only about building prompts, generating previews? Like I understand you can have a different sample prompt for each concept, but if you didn't care about that, does it actually change the way training works? Is there a different optimizer state for each concept or something?

Because right now, I've just combined all of my subjects into one folder, used [filewords], with files all manually captioned by myself with accurate descriptions, chosen appropriate training settings, and it does indeed train all of the subjects together, without needing to separate them out into the multiple concepts tab or a concepts list file.

Just wondering what specifically is happening with those options.

**Comment by kohweiteck** [comment]
I assume that for example , you can train multiple concepts like for DnD , you can train tortle monk ,tortle fighter, tabaxi assassin , kenku illusionist, etc... but one question remained, does we still need Instance Token as trigger word during training ? If so , what would it be? Take my example above as example.

**Reply by morphinapg** [reply]
Well I believe instance token is only helpful if you haven't manually tagged every photo. If you manually tag, you will already be using the keywords you plan to use in the photo captions, so [filewords] is enough there. 

But that's why I also ask my question, because if every photo is already tagged with all the relevant descriptions, including the keywords I want to use, what use do the extra concept tabs have? Or using a concepts list? It seems to me those would be useful if you need to build prompts, if you want to have different sample previews for different concepts, or if you just don't want to copy your different images into the same folder

But if I already have a single folder with all of my images, all manually tagged with the correct descriptions and keywords, then I feel like there's probably no reason to use the different concepts, either with the tabs or concept list json. 

The only exception I believe would be if each concept has a different optimizer momentum or something, but I would imagine that probably wouldn't be that useful for training a single model, so I doubt that's the way it works.
